ui/src/app/utils/access-utils.tsx

```diff
@@ -96,21 +96,6 @@ export const redirectToTwoFactorSetup = (): void => {
   window.open(getTwoFactorSetupUrl(), '_blank');
 };
 
-export const NIH_CALLBACK_PATH = '/nih-callback';
-
-export const redirectToNiH = (): void => {
-  AnalyticsTracker.Registration.ERACommons();
-
-  const url =
-    serverConfigStore.get().config.rasLogoutUrl +
-    serverConfigStore.get().config.shibbolethUiBaseUrl +
```

Comment:
only one reference to `shibbolethUiBaseUrl` remains in the UI: defaultServerConfig